packages/suite/src/views/wallet/tokens/common/TokensTable/TokenRow.tsx

```diff
@@ -108,8 +108,11 @@ export const TokenRow = ({
         account.symbol,
         token.contract,
     );
+
+    const explorer = useSelector(state => selectExplorer(state, network.symbol)) as Explorer;
+
     const coingeckoId = getCoingeckoId(account.symbol);
-    const explorerUrl = useExternalLink(getTokenExplorerUrl(network, token));
+    const explorerUrl = useExternalLink(getTokenExplorerUrl(explorer, network.networkType, token));
```

Comment:
Does not work here

```
             {
                                            label: <Translation id="TR_VIEW_IN_EXPLORER" />,
                                            icon: 'arrowUpRight',
                                            onClick: () => {
                                                window.open(explorerUrl, '_blank');
                                            },
                                        },
```
There is probably something wrong with this line. Same in NftsRow

Reply:
It takes me to 
<img width="1104" alt="Screenshot 2025-04-24 at 11 39 29" src="https://github.com/user-attachments/assets/e8db884e-9034-4a61-963b-4a231fd057ff" />